content/en/users/workload-manager/_index.md

```diff
@@ -4,9 +4,463 @@ linkTitle: "Workload Manager"
+
+## How to access the EGI Workload Manager service
+
+There are several options to access the service:
+
+1. Members of a scientific community whose resources pool is already configured
+   in the DIRAC4EGI instance > can use the
+   [DIRAC4EGI web portal](https://dirac.egi.eu:9443/DIRAC/)
+   to access the service, or use
+   [DIRAC Client](https://wiki.egi.eu/wiki/HOWTO22#Using_DIRAC_through_the_client).
+1. Individual researchers who want to do some number crunching for a limited
+   period of time, with a reasonable (not too high) number of CPUs > can use
+   the catch-all VO resource pool (vo.access.egi.eu). Submit a request for this
+   (CST to check identity and justification of use).
+1. Representatives of a community who want to try DIRAC and EGI > Same as #2.
+1. Representative of a community who wants to request DIRAC for the community's
+   own resource pool > Submit a request via the Marketplace, CST to call back
+   and discuss the details (which resource pool, what number of users, AAI,
+   etc.).
+
+## Getting Started
+
+### Submit a service order via the Marketplace
+
+User can request access to the service submitting a service-order to use the EGI
+HTC service directly from the EOSC or the EGI Marketplace:
+
+EOSC Marketplace:
+[https://marketplace.eosc-portal.eu/services/egi-workload-manager](https://marketplace.eosc-portal.eu/services/egi-workload-manager)
+
+EGI Marketplace:
+[https://marketplace.egi.eu/compute/73-workload-manager.html](https://marketplace.egi.eu/compute/73-workload-manager.html)
+
+Service orders are usually handled within 5 working days by the EGI User Support
+Team on shift.
+
+### Before starting
+
+#### Apply for your user credentials
+
+DIRAC uses X.509 certificates to identify and authenticate users. These
+certificates are delivered to each individual by trusted certification
+authorities.  
+If you have a personal certificate issued by a
+[EUGridPMA-certified](http://www.eugridpma.org/)
+authority you can use it for this tutorial. Otherwise, IHEP certification
+authority can deliver one for you. Click on the link `Online for User Request`
+on this
+[page](https://ihepca.ihep.ac.cn/reqForUser.html)
+and follow the instructions. Your certificate may take a few days to
+be delivered, so please ask for your certificate well in advance and in any
+case, before the tutorial starts.
+
+#### Export your credentials from your browser
+
+Your personal certificate is usually delivered to you via a web site and is
+automatically loaded in your browser.  You need to export it from the browser
+and put it in the appropriate format for DIRAC to use. This is a one-time
+operation. Please follow the instructions in this
+[document](http://www.chosensecurity.com/stuff/contentmgr/files/0/6d2fc0a807106169f266443672dc9ef2/document/web_browsers_with_certificates___chosensecurity.pdf)
+to export your certificate to a local file and named `$HOME/MyCertificate`.
+
+#### Install your credentials
```

Comment:
same as before, https://docs.egi.eu/users/check-in/voms/

Reply:
e8e23af546c28e3440fe4e174f543cebc4030ab9